docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -0,0 +1,162 @@
+
+The following example demonstrates how to create a tablespace in which the value of `min_num_replicas` does not correspond to `num_replicas`:
+
+```sql
+CREATE TABLESPACE us_east_tablespace 
+WITH (replica_placement='{"num_replicas": 5, "placement_blocks":
+[{"cloud":"aws","region":"us-east","zone":"us-east-1a","min_num_replicas":1},
+{"cloud":"aws","region":"us-east","zone":"us-east-1b","min_num_replicas":1},
+{"cloud":"aws","region":"us-east","zone":"us-east-1c","min_num_replicas":1}]}');
+```
+
+In the preceding example, the three `min_num_replicas` fields sum up to 3, whereas the total required replication factor is 5. As a result, even though `us_east_tablespace` has been created successfully, a notice is displayed after the execution stating that the additional two replicas are to be placed at the YB-Load balancer’s discretion in  `aws.us-east.us-east-1a`, `aws.us-east.us-east-1b`, and `aws.us-east.us-east-1c`. Based on the load, `aws.us-east.us-east-1a` may have three replicas and the other zones may have one, or it might later change such that `aws.us-east.us-east-1a` and `aws.us-east.us-east-1b` have two replicas each, whereas `aws.us-east.us-east-1c` has one replica. YB-Load balancer always honours the value of `min_num_replicas` and have at least one replica in each cloud.region.zone, and the additional replicas may be moved around based on the cluster load. 
+
+### How to Create a Single-Zone Tablespace
+
+To highlight the difference between multi-zone and single-zone tablespaces, the following example shows how to create a single-zone tablespace:
+
+```sql
+CREATE TABLESPACE us_west_1a_tablespace 
+WITH (replica_placement='{"num_replicas": 3, "placement_blocks":
+[{"cloud":"aws","region":"us-west","zone":"us-west-1a","min_num_replicas":3}]}');
+```
+
+If you create a table that uses `us_west_1a_tablespace`, each tablet of the table will have three replicas, but unlike multi-zone tablespaces, all three replicas for any tables and indexes in this tablespaces will be placed within the `us-west-1a` zone.
+
+## Creating Tables and Indexes in Tablespaces
+
+You can associate new tables and indexes with a corresponding tablespace. This defines the replication factor of the table or index. It also defines how the replicas of the table or index are to be spread across cloud, regions, and zones.
+
+### How to Use a Multi-Zone Tablespace
+
+Using the multi-zone tablespaces defined in [How to Create a Multi-Zone Tablespace](how-to-create-a-multi-zone-tablespace), you can apply the YSQL's `TABLESPACE` option to `CREATE TABLE` and `CREATE INDEX` statements, as follows:
+
+```sql
+CREATE TABLE employees (
+  employee_no integer PRIMARY KEY,
+  name text,
+  department text,
+  change_date date
+)
+TABLESPACE us_west_tablespace;
+```
+
+```sql
```

Comment:
Maybe we can stress on the point that a table and an index can be created in separate tablespaces.
Another point I would like to mention is that by default indexes will be placed according to cluster configuration.
Third point - can we mention in the partitions doc (not here) that a partition table will not inherit tablespaces from its parent? A partition table by default will be placed according to cluster configuration. If you don't want to fix it in context of this change, a GitHub issue would also be nice, maybe I can fix that later myself too.

Reply:
Added point 1 and 2 to tablespaces.md

Reply:
WRT point 3, I'm going to add it to the partitions.md, but I need to do it in a separate PR (I'll create a ticket right away, merge the PR, but you could always create a second PR for it. Another option is to add more information on partitions to the Develop section later on)

Reply:
Created https://github.com/yugabyte/yugabyte-db/issues/7910 to address point 3.